spec/models/timecode_spec.rb

```diff
@@ -1,21 +1,27 @@
 require 'rails_helper'
```

Comment:
https://github.com/rubizza-camp/Proconf/blob/add-statuses-to-episode/spec/models/episode_spec.rb

Reply:
@AndreyKorol what this comment for? You can just write a simple example after outlined a problem.